README.md

```diff
@@ -1,2 +1,146 @@
-# RoR-Budget-app
-The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.  
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+ - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 RoR-Budget-app <a name="about-project"></a>
```

Comment:
- [ ] [REQUIRED] A part of the project requirement states that you should record a video presenting the features of the project. The maximum length of the video is 5 minutes. Kindly ensure the video contains a clear and objective description of the project and the video should contain a demo of the project, highlighting all the project features. Lastly, the video should contain a highlight of a piece of the code you are proud of making.

- [ ] [REQUIRED] Also, ensure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance] the author of the original design as required by the [Creative Commons license of the design.
- [ ] [REQUIRED] Kindly deploy the application using Render, Netlify or GitHub Pages as stated in the project requiremnt.

Reply:
@Whoistolu 
thanks for the good and valuable feedback,
I try my best to apply the suggestions you gave, 
![Capture](https://github.com/mrkamin/RoR-Budget-app/assets/108926875/610451e0-7116-4b12-b66b-a7cdd264345c)